test/monitor.js

```diff
@@ -1018,7 +1027,7 @@ describe('Monitor', () => {
 
                         expect(res.statusCode).to.equal(200);
                         // account for hapi15 differences
-                        expect(out.data.length).to.be.greaterThan(0);
+                        expect(out.data.length).to.be.greaterThan(1);
```

Comment:
@lostthetrail I think this should be set to `least`. However; thats just to make it pass. Not entirely sure why one can't assert one event given you only make one wreck request in this single test case 🤔